rules/camelCaseLocalFunctionsRule.ts

```diff
@@ -13,11 +13,17 @@ class CamelCaseFunctionsWalker extends Lint.RuleWalker {
 		if (this.isIdentifier(name)) {
 			const firstLetter = name.text.charAt(0);
 			if (firstLetter !== firstLetter.toLowerCase()) {
+				const start = name.getStart(this.getSourceFile());
+				const width = name.getWidth(this.getSourceFile());
+				const fix = new Lint.Fix('camel-case-local-functions', [
+					this.createReplacement(start, 1, firstLetter.toLocaleLowerCase())
```

Comment:
We should pick one between toLowerCase/toLocaleLowerCase and use it in both places